Themes/default/languages/Admin.english.php

```diff
@@ -551,6 +551,7 @@
 $txt['enableEmbeddedFlash_warning'] = 'may be a security risk!';
 $txt['enableSpellChecking'] = 'Enable spell checking';
 $txt['enableSpellChecking_warning'] = 'this does not work on all servers.';
```

Comment:
$txt['enableSpellChecking_available'] = "This is available on your server.";

Reply:
Need we any message when it's available?